gst/mqtt/mqttsink.c

```diff
@@ -1067,6 +1120,92 @@ gst_mqtt_sink_set_mqtt_qos (GstMqttSink * self, const gint qos)
   self->mqtt_qos = qos;
 }
 
+/**
+ * @brief Getter for the 'ntp-sync' property.
+ */
+static gboolean
+gst_mqtt_sink_get_mqtt_ntp_sync (GstMqttSink * self)
+{
+  return self->mqtt_ntp_sync;
+}
+
+/**
+ * @brief Setter for the 'ntp-sync' property
+ */
+static void
+gst_mqtt_sink_set_mqtt_ntp_sync (GstMqttSink * self, const gboolean flag)
+{
+  self->mqtt_ntp_sync = flag;
+}
+
+/**
+ * @brief Getter for the 'ntp-srvs' property.
+ */
+static gchar *
+gst_mqtt_sink_get_mqtt_ntp_srvs (GstMqttSink * self)
+{
+  return self->mqtt_ntp_srvs;
+}
+
+/**
+ * @brief Setter for the 'ntp-srvs' property
+ */
+static void
+gst_mqtt_sink_set_mqtt_ntp_srvs (GstMqttSink * self, const gchar * pairs)
+{
+  gchar **pair_arrs = NULL;
+  guint hnum = 0;
+  gchar *pair;
+  guint i;
+
+  if (g_strcmp0 (self->mqtt_ntp_srvs, pairs) == 0)
+    return;
+
+  self->mqtt_ntp_srvs = g_strdup (pairs);
```

Comment:
Is it okay not to free `self->mqtt_ntp_srvs` before assignment?

Reply:
@dongju-chae Right!